layouts/shortcodes/toc-tree.html

```diff
@@ -11,7 +11,7 @@
   <ul>
     {{ range .sect.GroupBy "Weight" }}
       {{ range .ByTitle }}
-      {{ if not .Params.geekdocHidden }}
+      {{ if or (not .Params.geekdocHidden) (not (.Params.geekdocHiddenInPage | default true )) }}
```

Comment:
Just a small formatting/naming adjustment `{{ if or (not .Params.geekdocHidden) (not (default true .Params.geekdocHiddenTocTree)) }}`